frontends/web/src/locales/en/app.json

```diff
@@ -64,12 +64,13 @@
   },
   "aopp": {
     "addressRequest": "{{host}} is requesting a receiving address.",
+    "addressRequestWithLogo": "is requesting a receiving address",
     "banner": "Address request in progress.",
     "errorTitle": "Error during address request ",
     "labelAddress": "Address",
     "labelMessage": "Message",
     "reverifyInfoText": "Use to verify agianst external service",
-    "signing": "To proceed, sign message on your BitBox02.",
+    "signing": "To proceed, sign message on your BitBox02",
```

Comment:
please change existing text on locize only. it will land in the repo with the next locize-pull (i do it regularly before releases and on-demand ;)). it is confusing to always do it in both places imho.

Reply:
I know but this way we have nicer commits and I don't forget.

Reply:
Just a reminder, I have forgotten text changes in the past i.e. "Buy {name}" instead of "Buy {coinName}" or similar.

Should I undo it, here now in this PR so you can pull later?

Reply:
ack, no need to undo